src/applications/mhv-secure-messaging/tests/e2e/pages/GeneralFunctionsPage.js

```diff
@@ -96,6 +97,21 @@ class GeneralFunctionsPage {
       Math.random() * (now.getTime() - currentDate.getTime());
     return new Date(randomTime).toISOString();
   };
+
+  getParsedDate = date => {
+    let year = format(date, 'yyyy');
```

Comment:
I would recommend stick with using `date-fns` function `getYear` since we are already using that lib here
```suggestion
    const year = getYear(date);
```